packages/cli-kit/src/node/node-package-manager.ts

```diff
@@ -183,6 +186,19 @@ export async function getDependencies(packageJsonPath: string): Promise<{[key: s
   return {...dependencies, ...devDependencies}
 }
 
+/**
+ * Returns true if the app uses workspaces, false otherwise.
+ * @param packageJsonPath {string} Path to the package.json file
+ * @param pnpmWorkspacePath {string} Path to the pnpm-workspace.yaml file
+ * @returns A promise that resolves with true if the app uses workspaces, false otherwise.
+ */
+
+export async function usesWorkspaces(appDirectory: string): Promise<boolean> {
```

Comment:
What about adding a test to cover the two scenarios here?

Reply:
I've added integration tests that cover it, but I'll also add unit tests as you suggested.